docker-builds/reference-crucible/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM phusion/passenger-full:0.9.18
+MAINTAINER Erik Wiffin
+
+# Enable nginx and passenger
+RUN rm -f /etc/service/nginx/down
+
+### Our Stuff ###
+# Update Apt
+RUN sh -c "echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' >> /etc/apt/sources.list" && \
+    apt-key adv --keyserver keyserver.ubuntu.com --recv 7F0CEB10 && \
+    apt-get update
+
+# Install MongoDB
+RUN apt-get install -y mongodb mongodb-server && \
```

Comment:
Yeah, it's nice to be able to look at the docker-compose file and see "these are the services you'll be interacting with".